docs/docs.md

```diff
@@ -29,6 +32,9 @@ Create a Dynamics client from environment variables:
 3. DYNAMICS_CLIENT_ID
 4. DYNAMICS_CLIENT_SECRET
 5. DYNAMICS_SCOPE
+6. *DYNAMICS_RESOURCE (Optional)*
+
+*If you are experiencing auth errors, inspect the returned auth token's `aud` and see whether it resolves to `00000002-0000-0000-c000-000000000000` instead of your CRM url. If so, you may want to specify the resource URL using the optional 'DYNAMICS_RESOURCE' environment variable.*
```

Comment:
See [this comment](https://github.com/MrThearMan/dynamics-client/pull/4/files#r1148501377).